unison-runtime/src/Unison/Runtime/Foreign.hs

```diff
@@ -19,6 +20,8 @@ module Unison.Runtime.Foreign
   )
 where
 
+-- import GHC.Exts (unsafePtrEquality#, isTrue#, Any)
```

Comment:
If these aren't actually being used, can we get rid of them and `LANGUAGE MagicHash`?